src/en/components/error-summary/use-case.md

```diff
@@ -9,7 +9,7 @@ eleventyNavigation:
   otherNames: error overview.
   description: A list of user errors on a page or in a flow.
   thumbnail: /images/common/components/preview-error-summary.svg
-  alt: A red outlined box holds one thick grey line above a stack of  three smaller thick red lines representing a heading and text.
+  alt: A red rectangle frames a prominent heading that reads "There's a problem" and a numbered list of 3 links with error summary items in the link text.
```

Comment:
Would this alt text be accurate since the preview images are just low fidelity versions of the component?

Reply:
Good catch, I'll update that one!